rotkehlchen/chain/ethereum/uniswap/uniswap.py

```diff
@@ -220,6 +244,224 @@ def _get_known_asset_price(
 
         return asset_price
 
+    def _get_trades(
+            self,
+            addresses: List[ChecksumEthAddress],
+            from_timestamp: Timestamp,
+            to_timestamp: Timestamp,
+            db_query_get_used_query_range: Callable,
+            db_query_update_used_query_range: Callable,
+            db_query_get_trades: Callable,
+            db_query_add_trades: Callable,
+            get_ammtrade_from_trade: Callable,
+    ) -> AddressTrades:
+        """Request via graph all trades for new addresses and the latest ones
+        for already existing addresses. Then the requested trade are stored in
+        DB and finally all DB trades are returned.
+
+        Requested trades are managed as <AMMTrade>, whilst trades read from DB
+        and insterted to DB are managed as <Trade> (with a custom identifer).
```

Comment:
insterted -> inserted

read from DB and insterted to DB -> read/written to/from DB